lib/utils/networks.js

```diff
@@ -146,6 +134,10 @@ export const chainIdToName = (chainId) => {
   return Object.values(NETWORK_DATA).find((network) => network.chainId === chainId)?.name
 }
 
+export const chainIdToView = (chainId) => {
+  return Object.values(NETWORK_DATA).find((network) => network.chainId === chainId)?.view
```

Comment:
So great to have a list of all chains in here so when someone connects with say Avalanche or Arbitrum it will say "You're connected to Arbitrum" instead of "You're connected to Unknown network" 👍  👍